LICENSE.md

```diff
@@ -0,0 +1,19 @@
+Copyright 2017 Chris Carpita
```

Comment:
I've labelled as MIT license recently (see README) to clarify rights for contributors, so this file should be consistent.  Thanks for adding though!